index.js

```diff
@@ -239,14 +240,25 @@ const plugin = fp(async function (app, opts) {
     }
   }
 
-  fastifyGraphQl.extendSchema = fastifyGraphQl.extendSchema || function (s) {
-    if (typeof s === 'string') {
-      s = parse(s)
-    } else if (!s || typeof s !== 'object') {
-      throw new MER_ERR_INVALID_OPTS('Must provide valid Document AST')
+  const initialExtendSchema = fastifyGraphQl.extendSchema
+  fastifyGraphQl.extendSchema = (s) => {
+    if (initialExtendSchema) {
+      initialExtendSchema(s)
```

Comment:
I'm trying to figure out how to test this line, currently not covered. In the meantime I opened the PR to receive feedbacks.